aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,18 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## September 6 - September 17, 2021
+- Enhanced List to support vertical alignment and added guidance for [multi-line lists](/templates/lists).
```

Comment:
Enhanced List to support `top-alignment`.....

Reply:
Updated in latest commit